.github/workflows/selfhosted.yml

```diff
@@ -0,0 +1,120 @@
+# CI workflow that runs on self hosted cluster
+name: Self-hosted rules
+
+on:
+  workflow_call:
+    inputs:
+      used-labels:
+        description: 'Label names in JSON format'
+        required: true
+        type: string
```

Comment:
Can't you just hard-code this? To simplify the workflow "signature"

Reply:
I don't think we can hardcode this because the label name is different for each platform, as it defines the name of the queue that the workflow will be submitted. For example the `run-on-tii1q_b1` label will submit this workflow on the queue named tii1q_b1.

Reply:
Ok, I see. I thought it would have been always something like [`run-on-sim`](https://github.com/qiboteam/qibo/issues?q=is%3Aopen+label%3Arun-on-sim
+sort%3Aupdated-desc).

You can resolve the conversation.